src/main/play-doc/developer/ConductrSandboxCluster.md

```diff
@@ -0,0 +1,70 @@
+# ConductR sandbox cluster
```

Comment:
Remove the word "cluster" - the emphasis is just on managing ConductR. In fact, we're only providing the single node version for development; we may provide a cluster-able version in the future, but not right now.